### PR TITLE
VL-159 Add retry-after value

### PR DESCRIPTION
**Pull request by datalek**
#### Motivation and Context

Add the value `retry-after` and `retryAfter` returned by two endpoint ([/delivery/requests](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/pagopa/pn-delivery/develop/docs/openapi/api-external-b2b-pa-v1.yaml#/SenderReadB2B/retrieveNotificationRequestStatus) and [/delivery-progresses/streams/{streamId}/events](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/pagopa/pn-delivery-push/develop/docs/openapi/api-external-b2b-webhook-v1.yaml#/Events/consumeEventStream))

#### How Has This Been Tested?

Unit and e2e tests

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/domain/CheckNotificationStatusRecord.ts [hunk 1]
```diff
@@ -49,7 +49,10 @@ export const makeCheckNotificationStatusRecord =
                 t.exact(NewNotificationRequestStatusResponse).encode({ ...n, notificationRequestStatus: 'ACCEPTED' })
             )
           ),
-          O.map((response) => ({ statusCode: 200 as const, returned: response })),
+          O.map((response) => ({
+            statusCode: 200 as const,
+            returned: { ...response, retryAfter: env.retryAfterMs / 1000 },
```
Comment (kin0992) [inline src/domain/CheckNotificationStatusRecord.ts:54]:
```suggestion
            returned: { ...response, retryAfter: env.retryAfterMs },
```
Not sure we should divide by 1000. According to the doc:
> retry-after: Numero di millisecondi di attesa prima di effettuare una nuova lettura di eventi.

so I guess it should be `env.retryAfterMs`.

**Review by kin0992 — CHANGES_REQUESTED** [review]

File: src/domain/CheckNotificationStatusRecord.ts (hunk 1 above)
Comment (datalek) [reply src/domain/CheckNotificationStatusRecord.ts:54]: There are two different *retry after* field, one in second and the other one in millisecond. This [one](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/pagopa/pn-delivery/develop/docs/openapi/api-external-b2b-pa-v1.yaml#/SenderReadB2B/retrieveNotificationRequestStatus) is in second and this [one](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/pagopa/pn-delivery-push/develop/docs/openapi/api-external-b2b-webhook-v1.yaml#/Events/consumeEventStream) in millisecond

Comment (kin0992) [reply src/domain/CheckNotificationStatusRecord.ts:54]: That's right! My bad!

**Review by kin0992 — APPROVED** [review]
🦘

**Pull request merged**